mscore/prefsdialog.ui

```diff
@@ -3453,6 +3453,49 @@ Adjusting latency can help synchronize your MIDI hardware with MuseScore's inter
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="resetPositionsOnMuseScore1And2Import">
+         <property name="title">
+          <string>Reset Positions on MuseScore 1 and 2 Import</string>
```

Comment:
Please change to:

Reset element positions when importing MuseScore 1 and 2 files

Reply:
Or "Reset Element Positions on Importing MuseScore 1 and 2 Scores"

Reply:
I really prefer “when”. I'm OK with either “files” or “scores”.

But regardless, the capitalization should be changed (capitalize only “Reset” and “MuseScore”), for consistency with the rest of the UI.

Reply:
No, it is and needs to be Title Case, exactly for the reason of consistency

Reply:
Anyway, izt is way too long, so I shortened it to "Reset Element Positions" and added a ToolTip with the longer explanation (and in Sentence case)